main/src/main/scala/sbt/Keys.scala

```diff
@@ -491,6 +491,9 @@ object Keys {
   private[sbt] val postProgressReports = settingKey[Unit]("Internally used to modify logger.").withRank(DTask)
   @deprecated("No longer used", "1.3.0")
   private[sbt] val executeProgress = settingKey[State => TaskProgress]("Experimental task execution listener.").withRank(DTask)
+  val lintBuild = inputKey[Unit]("Stop a background job by providing its ID.")
```

Comment:
Did you forget to change the description?

Reply:
addressed in 460d1f5.